### ISSUE TITLE
How to train custom model with my own dataset from scratch(without using any pretrained weights e.g coco, imagenet)?

### ISSUE THREAD

**Issue by mudasar477** [matterport/Mask_RCNN#1317]

**Comment by adions025** [comment]
hi,
i have the same question, if you have something can you explain me how did you do it?

**Comment by May-forever** [comment]
Dear mudasar477,

I want to use Mask-RCNN to train on my custom dataset (just one class).

First of all, I use Labelme to create groundtruth for each image. Each image has a .json file as its ground-truth.

After that, I don't know what should I do...

I want to know how to convert my groundtruth file to a format that Mask-rcnn can use. Do you have any tools ?

Looking forward to hearing from you, thanks in advance.

With best regards,
Ming

**Comment by josephofiowa** [comment]
@mudasar477 Yes, you can do this. For the your image dataset, it's recommended you use the same annotation tool as the balloons tutorial: http://www.robots.ox.ac.uk/~vgg/software/via/ For your custom weights, I recommend using the pre-trained weights from COCO to decrease your training time. You can start from scratch however.

Email me if you have questions getting setup: joseph@roboflow.ai

**Comment by LoneBekkeheien** [comment]
@mudasar477 , did you figure out how to train from scratch?
